zetta_utils/mazepa_layer_processing/common/volumetric_callable_operation.py

```diff
@@ -94,6 +94,12 @@ def __call__(  # pylint: disable=keyword-arg-before-vararg
                     semaphore_stack.enter_context(semaphore(semaphore_type))
             result_raw = self.fn(**task_kwargs)
             torch.cuda.empty_cache()
+
+        # If no destination layer, we can bail out now.  Possibly we
+        # could bail out a bit sooner.  ToDo: study this more.
+        if dst is None:
+            return
```

Comment:
could you overwrite it as `if dst is not None: ... # rest of the code`
as a rule of thumb we try to minimize the number of return statements in our functions